gulp/html.js

```diff
@@ -321,12 +321,12 @@ module.exports = (config, cb) => {
     // )
 
     // Format
-    .pipe(
-      prettify({
-        indent_with_tabs: false,
-        max_preserve_newlines: 1
-      })
-    )
+    // .pipe(
```

Comment:
Not directly related, but this was also "unminifying" the minified preload polyfill added below. And it was slow.